core/spec/models/spree/exchange_spec.rb

```diff
@@ -72,7 +72,7 @@ module Spree
     end
 
     describe ".model_name" do # for dom_id
```

Comment:
as we no longer implement model_name ourselves I think you can safely kill this spec now